src/cmd-build

```diff
@@ -247,7 +247,10 @@ cat > "${commitmeta_input_json}" <<EOF
 }
 EOF
 
-prepare_git_artifacts "${configdir_gitrepo}" "${PWD}/coreos-assembler-config.tar.gz" "${PWD}/coreos-assembler-config-git.json"
+if [ -d "${workdir}/src/yumrepos" ]; then
```

Comment:
Maybe we should consolidate on a single config.json file here as I'm introducing for the variant support?